packages/paste-core/layout/grid/src/utils.tsx

```diff
@@ -13,33 +12,30 @@ import {
 type Vertical = ResponsiveValue<boolean>;
 
 // Gets the gutter and returns the value to be used as negative margin to Grid
-export const getOuterGutterPull = (theme: ThemeShape, gutter?: Space): Margin => {
+export const getOuterGutterPull = (gutter?: Space): Margin => {
   if (Array.isArray(gutter)) {
     return (gutter as SpaceOptions[]).map((value: SpaceOptions) => {
-      return `-${theme.space[value]}` as SpaceOptions;
+      if (value !== 'space0') {
+        return value.replace('space', 'spaceNegative') as SpaceOptions;
+      }
+      if (value === 'space0') {
+        return 'space0' as SpaceOptions;
+      }
+      return null;
     });
   }
 
-  if (gutter) {
-    return `-${theme.space[gutter as SpaceOptions]}` as SpaceOptions;
-  }
-
-  return 'auto';
-};
-
-// Gets the gutter and returns the value to be used as padding for Columns
-export const getColumnGutters = (theme: ThemeShape, gutter?: Space): Padding => {
-  if (Array.isArray(gutter)) {
-    return (gutter as SpaceOptions[]).map((value: SpaceOptions) => {
-      return `${theme.space[value]}` as SpaceOptions;
-    });
+  if (gutter && gutter !== 'space0') {
+    // Ignoring this ts error becuase we're checking for space0 already.
+    // @ts-ignore
+    return gutter.replace('space', 'spaceNegative') as SpaceOptions;
   }
 
-  if (gutter) {
-    return `${theme.space[gutter as SpaceOptions]}` as SpaceOptions;
+  if (gutter === 'space0') {
```

Comment:
nit: move this `if` clause above the previous one and make the above one just `if (gutter)`

Would that fix the TS issue? 🤔

Reply:
Looks like that doesn't help with the TS error.